styles.css

```diff
@@ -12,7 +12,10 @@ Error: #EFB7EC */
 * {
 margin: 0;
 font-family: 'Montserrat', sans-serif;
-color: #FFF;
+}
+
+body {
+  color: #FFF;
 }
 
 /* ---------HEADER-------------- */
```

Comment:
On line 15, is changing this color property going to affect our other text now that it is not applied to all?

Reply:
I think we have color properties being added to the proper containers currently